### ISSUE TITLE
[Extension Bug] Docker manage images doesn't work

### ISSUE THREAD

**Issue by Crocmagnon** [raycast/extensions#583]
# Extension – [Docker](https://github.com/raycast/extensions/commits/main/extensions/docker)

Author: @priithaamer 

## Description
The manage images command of the docker extension fails with a generic error message.

<img width="768" alt="Screenshot 2021-12-31 at 12 31 26" src="https://user-images.githubusercontent.com/3630554/147821105-af1663bd-585d-466b-8efb-656c5d95b721.png">

## Steps To Reproduce

1. Install docker extension
2. Run the manage image command
3. It fails

## The current behavior
Generic error message (see screenshot above)

```
TypeError: Cannot read properties of null (reading 'join')

xs:image_list.js:20:618

---
17: `),Cl=e=>e.join(" "),kl=e=>e==null?"-":Array.isArray(e)?e.join(" "):e,fn=(e,t=2)=>{if(e===0)return"0 Bytes";let r=1024,n=t<0?0:t,i=["Bytes","KB","MB","GB","TB","PB","EB","ZB","YB"],o=Math.floor(Math.log(e)/Math.log(r));return parseFloat((e/Math.pow(r,o)).toFixed(n))+" "+i[o]};var pe=Y(require("@raycast/api")),Xt=Y(require("react"));var ln=e=>e.errno===-61&&e.code==="ECONNREFUSED";function cn({error:e}){let[t,r]=(0,Xt.useState)([]);(0,Xt.useEffect)(()=>{async function i(){let o=await(0,pe.getApplications)();r(o)}i()},[]);let n=t.find(({bundleId:i})=>i==="com.docker.docker");return _jsx(pe.Detail,{markdown:Tl(e),actions:ln(e)&&n!==void 0?_jsx(pe.ActionPanel,null,_jsx(pe.OpenAction,{title:"Launch Docker",target:n.path})):null})}var Tl=e=>{let t="Error message:\n\n```\n"+e.message+"\n```";return ln(e)?["## \u26A0\uFE0F Error connecting to Docker",t].join(`
18: `):`## An Error Occurred:
19: 
20: ${t}`};var ks=Y(nn()),Ts=Y(require("@raycast/api")),Rs=Y(require("react"));function dn({imageId:e}){let t=(0,Rs.useMemo)(()=>new ks.default,[]),{useImageInfo:r}=Zt(t),{imageInfo:n,isLoading:i}=r({Id:e});return _jsx(Ts.Detail,{navigationTitle:"Image Details",isLoading:i,markdown:Cs(n)})}var Ue=Y(require("@raycast/api")),Ps=({action:e,onSuccess:t,onFailure:r})=>async()=>{try{await e(),t!==void 0&&await(0,Ue.showToast)(Ue.ToastStyle.Success,...Os(t()))}catch(n){n instanceof Error&&r!==void 0&&await(0,Ue.showToast)(Ue.ToastStyle.Failure,...Os(r(n)))}},Os=e=>Array.isArray(e)?e:[e];function xs(){let e=(0,Is.useMemo)(()=>new As.default,[]),{useImages:t}=Zt(e),{images:r,isLoading:n,error:i,removeImage:o}=t();return i?_jsx(cn,{error:i}):_jsx(N.List,{isLoading:n},r?.map(s=>_jsx(N.List.Item,{key:s.Id,title:dt(s),icon:{source:"icon-image.png",tintColor:N.Color.SecondaryText},accessoryTitle:fn(s.Size)??"",actions:_jsx(N.ActionPanel,{title:dt(s)},_jsx(N.PushAction,{title:"Inspect",icon:{source:N.Icon.Binoculars},shortcut:{modifiers:["cmd"],key:"i"},target:_jsx(dn,{imageId:s.Id})}),_jsx(N.ActionPanel.Item,{title:"Remove Image",icon:{source:N.Icon.Trash,tintColor:N.Color.Red},shortcut:{modifiers:["cmd","shift"],key:"x"},onAction:Ps({action:()=>o(s),onSuccess:()=>`Image ${dt(s)} removed`,onFailure:({message:a})=>a})}))})))}0&&(module.exports={});
21: /*! safe-buffer. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
22: 
---

    at ray-navigation-stack
$v:index.js:68:33481
Ol:index.js:68:33586
js:index.js:68:34196
k_:index.js:68:32030
    at ray-root
Xf:index.js:68:34500
```

## The expected behavior
I guess see the list of images

### Raycast version
Version: 1.26.3

**Comment by Crocmagnon** [comment]
.

**Comment by Crocmagnon** [comment]
.

**Comment by pernielsentikaer** [comment]
This should now be fixed @Crocmagnon in #1703
